.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+dist/*.min.js
+.DS_Store
```

Comment:
I've always been of the opinion that people should have this in their global gitignore.
Same with IDE config files.